### DISCUSSION TITLE
Automatically turn on when VPN is connected

### DISCUSSION THREAD

**Discussion by Dr-Ari-Gami** [Mygod/VPNHotspot#380] [Q&A]
Hey, I can't seem to figure out how to get VPN Hotspot to automatically enable itself when I turn on my VPN and hotspot. 

The option is there, I can enable `ap_be_wlan2` but it won't enable automatically.

Please advise,
Thank you!

**Comment by Mygod** [comment]
Use the monitor feature. (See top right in your app.)

**Reply by Dr-Ari-Gami** [reply]
I just tried that. It seems to reset upon device reboot. Am I doing something wrong?

**Reply by Mygod** [reply]
Make sure battery optimizations are disabled for this app.
